flaml/automl/task/generic_task.py

```diff
@@ -819,9 +819,14 @@ def default_estimator_list(self, estimator_list: List[str]) -> List[str]:
                     try:
                         import prophet
 
-                        estimator_list += ["prophet", "arima", "sarimax"]
+                        estimator_list += [
+                            "prophet",
+                            "arima",
+                            "sarimax",
+                            "holt-winters",
+                        ]
                     except ImportError:
-                        estimator_list += ["arima", "sarimax"]
+                        estimator_list += ["arima", "holt-winters"]
```

Comment:
Is there a good reason to remove "sarimax" here?

Reply:
This was a mistake, corrected